### PR TITLE
Replace h1 elements for h2 or h3 where applicable

### PR DESCRIPTION
**Pull request by JustinPinner**
## Why are you doing this?
Because there should only be one `<h1>` on a page. The showcase page contained nine!

[**Trello Card**](https://trello.com/c/2lLVUOPz/1786-only-one-h1-per-page)

## Changes
* Leave the main page title alone, it's an `<h1>` with multiple spans inside
* Change all other `<h1>` elements to `<h2>` or `<h3>` depending where they sit within the page hierarchy

## Screenshots
**Before**
screencapture-support-thegulocal-uk-before-2018-07-24-12_58_32

**After**
screencapture-support-thegulocal-uk-after-2018-07-24-12_21_00

Note: You shouldn't have seen any differences between those screen captures!

### CODE REVIEW

File: assets/components/heading/heading.jsx [hunk 1]
```diff
@@ -7,10 +7,10 @@ import React, { type Node } from 'react';
 
 // ----- Types ----- //
 
-export type HeadingRange = 1 | 2 | 3 | 4 | 5 | 6;
+export type HeadingSize = 1 | 2 | 3 | 4 | 5 | 6;
```
Comment (Ap0c) [inline assets/components/heading/heading.jsx]: 👍

File: assets/components/doubleHeading/doubleHeading.jsx [hunk 2]
```diff
@@ -22,14 +23,10 @@ export default function DoubleHeading(props: PropTypes) {
...
   return (
-    <div className={className}>
-      <h1 className="component-double-heading__heading">
-        { props.heading }
-      </h1>
-      <h2 className="component-double-heading__subheading">
-        { props.subheading }
-      </h2>
-    </div>
+    <Heading size={3} className={className}>
```
Comment (Ap0c) [inline assets/components/doubleHeading/doubleHeading.jsx]: Should this size be passed in as a prop to the `DoubleHeading` component?

File: assets/components/highlights/highlights.jsx [hunk 3]
```diff
@@ -23,13 +24,13 @@ export default function Highlights(props: PropTypes) {
   }
 
   return (
-    <h1 className={classNameWithModifiers('component-highlights', props.modifierClasses)}>
+    <Heading size={2} className={classNameWithModifiers('component-highlights', props.modifierClasses)}>
```
Comment (Ap0c) [inline assets/components/highlights/highlights.jsx]: As above.

File: assets/components/readyToSupport/readyToSupport.jsx [hunk 4]
```diff
@@ -22,10 +22,10 @@ export default function ReadyToSupport(props: PropTypes) {
   return (
     <section className="component-ready-to-support">
       <div className="component-ready-to-support__content">
-        <h1 className="component-ready-to-support__heading">
+        <Heading size={3} className="component-ready-to-support__heading">
```
Comment (Ap0c) [inline assets/components/readyToSupport/readyToSupport.jsx]: As above.

File: assets/components/featureList/featureList.jsx [hunk 5]
```diff
@@ -50,16 +47,13 @@ function FeatureList(props: PropTypes) {
...
   if (props.heading) {
     return (
-      <Heading
-        className="component-feature-list__heading"
-        size={props.size}
-      >
-        {props.heading}
-      </Heading>
+      <span className="component-feature-list__heading">
+        <strong>{props.heading}</strong>
```
Comment (Ap0c) [inline assets/components/featureList/featureList.jsx]: 👍

File: assets/components/whySupport/whySupport.jsx [hunk 6]
```diff
@@ -20,29 +20,29 @@ export default function WhySupport() {
   return (
     <div className="component-why-support">
       <PageSection heading="Why support?" modifierClass="why-support">
-        <h1 className="component-why-support__heading">
+        <Heading size={3} className="component-why-support__heading">
```
Comment (Ap0c) [inline assets/components/whySupport/whySupport.jsx]: As above.

File: assets/pages/subscriptions-landing/subscriptionsLanding.jsx [hunk 7]
```diff
@@ -43,12 +43,20 @@ const content = (
...
+          paperHeadingSize={3}
+          paperDigitalHeadingSize={3}
+        />
       </section>
-      <WhySupport />
-      <ReadyToSupport ctaUrl={`#${supporterSectionId}`} />
+      <WhySupport headingSize={3} />
+      <ReadyToSupport
+        ctaUrl={`#${supporterSectionId}`}
+        headingSize={3}
```
Comment (Ap0c) [inline assets/pages/subscriptions-landing/subscriptionsLanding.jsx]: Should this be `2`?

Comment (JustinPinner) [reply assets/pages/subscriptions-landing/subscriptionsLanding.jsx]: Yes :(

**Comment by JustinPinner** [comment]
I think we decided we were ignoring the `com.gu.identity:identity-crypto_2.12@3.87 › org.bouncycastle:bcprov-jdk15on@1.58` snyk warning because `identity` doesn't make use of the part of the `bcprov-jdk15on` API that is vulnerable.

**Comment by JustinPinner** [comment]
Turns out @johnduffell had fixed the snyk problem - just had to merge master. Ready to merge now...

**Pull request merged**